ProjectTemplates/AspNetCoreMicrosoftIdentityWebProjectTemplates.csproj

```diff
@@ -36,4 +36,11 @@
     <Content Include="templates\**\*" Exclude="templates\**\bin\**;templates\**\obj\**" />
     <Compile Remove="**\*" />
   </ItemGroup>
+
+  <ItemGroup>
```

Comment:
I thought I had provided a comment here, but it disappeared
These files should not be excluded.

This is due to a bug in Visual Studio (when you did a renaming)
Just remove these 6 lines (the ItemGroup)